examples/multiple_viewer_widget.py

```diff
@@ -0,0 +1,434 @@
+"""
+This is example how to have more than one viewer in the same napari instance.
+Additional viewers state will be synchronized with the main viewer and do not
```

Comment:
```suggestion
Additional viewers state will be synchronized with the main viewer
```
The end of the sentence seems to be missing :)